### DISCUSSION TITLE
How can I disable results of binary files

### DISCUSSION THREAD

**Discussion by IamSmeagol** [ibhagwan/fzf-lua#1654] [Q&A]
I saw that binary files are removed from the previewer, how can I remove them from the search results entirely?

**Comment by ibhagwan** [comment]
Look into that man pages of fd and rg and see if there’s an option to exclude binaries, once you find that customize the cmd, rg_opts or fd_opts.

**Reply by IamSmeagol** [reply]
I am using fzf and can't find anything in it's manual about hiding binary files. I saw that you already did [this](https://github.com/ibhagwan/fzf-lua/blob/12ac915c26ec6337ff96be0b8b2311736cca4c13/lua/fzf-lua/previewer/builtin.lua#L821), but wasn't sure if there was a way to apply it to my searches too. If needed I'll open a discussion on their repo instead. Thank you for the prompt reply

**Reply by ibhagwan** [reply]
> I am using fzf and can't find anything in it's manual about hiding binary files. I saw that you already did [this](https://github.com/ibhagwan/fzf-lua/blob/12ac915c26ec6337ff96be0b8b2311736cca4c13/lua/fzf-lua/previewer/builtin.lua#L821), but wasn't sure if there was a way to apply it to my searches too. If needed I'll open a discussion on their repo instead. Thank you for the prompt reply


You’re not using fzf for enumerating the files it’s just fuzzy matcher, the underlying command is rg or fd which you can see if you run `:FzfLua files debug=true` and you’ll see, the code you linked is for the previewer, you’ll need to explore this with rg or fd.
